src/VirtoCommerce.CatalogExportImportModule.Web/Scripts/module.js

```diff
@@ -7,13 +7,15 @@ if (AppDependencies !== undefined) {
     AppDependencies.push(moduleName);
 }
 
-angular.module(moduleName, ['ui.grid.autoFitColumns']).run(['virtoCommerce.catalogModule.catalogImportService', 'virtoCommerce.catalogModule.catalogExportService', function (catalogImportService, catalogExportService) {
+angular.module(moduleName, ['ui.grid.autoFitColumns']).run(['virtoCommerce.featureManagerSubscriber', 'virtoCommerce.catalogModule.catalogImportService', 'virtoCommerce.catalogModule.catalogExportService', function (featureManagerSubscriber, catalogImportService, catalogExportService) {
+
     catalogImportService.register({
         name: 'Description import',
         description: 'Descriptions data import from CSV',
         icon: 'fa fa-list-alt',
         controller: 'virtoCommerce.catalogExportImportModule.fileUploadController',
-        template: 'Modules/$(VirtoCommerce.CatalogExportImport)/Scripts/blades/file-upload.tpl.html'
+        template: 'Modules/$(VirtoCommerce.CatalogExportImport)/Scripts/blades/file-upload.tpl.html',
+        predefinedDataType: { key: 'Descriptions', value: 'EditorialReview' }
```

Comment:
maybe just 'dataType'?

Reply:
We are setting blade.dataType inside fileUploadController